<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>765</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>760</width>
    <height>765</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>760</width>
    <height>765</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Malgun Gothic</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Embulk</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupSrc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Source</string>
    </property>
    <widget class="QLineEdit" name="lineEditSrcHost">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>51</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelSrcHost">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Host IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSrcPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>81</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSrcPort">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>81</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelSrcDB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>111</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DB (SID)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSrcDB">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>111</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelSrcUser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>141</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>User</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSrcUser">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>141</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelSrcPasswd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>171</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditSrcPasswd">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>171</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelSrcRdbms">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>RDBMS</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxSrcDbms">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>21</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string>Oracle</string>
     </property>
     <item>
      <property name="text">
       <string>Oracle</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButtonSrcTestConn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>21</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Test Connection</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSrcConn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>51</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Connection</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupTgt">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>361</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>Target</string>
    </property>
    <widget class="QLineEdit" name="lineEditTgtHost">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>51</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelTgtHost">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>51</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Host IP</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTgtPort">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>81</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTgtPort">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>81</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelTgtDB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>111</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DB (SID)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTgtDB">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>111</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelTgtUser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>141</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>User</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTgtUser">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>141</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelTgtPasswd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>171</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditTgtPasswd">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>171</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelTgtRdbms">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>RDBMS</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxTgtDbms">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>21</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string>Oracle</string>
     </property>
     <item>
      <property name="text">
       <string>Oracle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MySQL</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButtonTgtTestConn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>21</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Test Connection</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupTableList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>721</width>
      <height>471</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTreeWidget" name="treeViewSrcTables">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>451</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
    <widget class="QPlainTextEdit" name="textEditQuery">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonConfirm">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>90</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPlus">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>170</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMinus">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>210</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="treeViewMigTables">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>120</y>
       <width>331</width>
       <height>341</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="labelQuery">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>0</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Query</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonOptions">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>700</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>700</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>메뉴</string>
    </property>
   </widget>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
